src/app/ReadHandler.cpp

```diff
@@ -108,8 +128,33 @@ CHIP_ERROR ReadHandler::OnStatusReport(Messaging::ExchangeContext * apExchangeCo
                  err = CHIP_ERROR_INVALID_ARGUMENT);
     switch (mState)
     {
+    case HandlerState::SubscribingResponding:
+        ClearInitialReport();
+        RefreshSubscribeSyncTimer();
```

Comment:
RefreshSubscribeSyncTimer returns a CHIP_ERROR. please do not silently ignore (at least log)